{
  "schema_version": "1.4.0",
  "id": "GHSA-qphh-5fv5-2mjj",
  "modified": "2022-05-17T04:49:45Z",
  "published": "2022-05-17T04:49:45Z",
  "aliases": [
    "CVE-2013-4196"
  ],
  "details": "The object manager implementation (objectmanager.py) in Plone 2.1 through 4.1, 4.2.x through 4.2.5, and 4.3.x through 4.3.1 does not properly restrict access to internal methods, which allows remote attackers to obtain sensitive information via a crafted request.",
  "severity": [

  ],
  "affected": [

  ],
  "references": [
    {
      "type": "ADVISORY",
      "url": "https://nvd.nist.gov/vuln/detail/CVE-2013-4196"
    },
    {
      "type": "WEB",
      "url": "https://bugzilla.redhat.com/show_bug.cgi?id=978475"
    },
    {
      "type": "WEB",
      "url": "http://plone.org/products/plone-hotfix/releases/20130618"
    },
    {
      "type": "WEB",
      "url": "http://plone.org/products/plone/security/advisories/20130618-announcement"
    },
    {
      "type": "WEB",
      "url": "http://seclists.org/oss-sec/2013/q3/261"
    }
  ],
  "database_specific": {
    "cwe_ids": [

    ],
    "severity": "MODERATE",
    "github_reviewed": false,
    "github_reviewed_at": null,
    "nvd_published_at": "2014-03-11T19:37:00Z"
  }
}